lib/src/accordion-group/types.ts

```diff
@@ -13,7 +13,7 @@ export type AccordionPropsType = {
    */
   label: string;
   /**
-   * Element or path used as the icon that will be placed next to panel label.
+   * Material Symbol name or SVG element used as the icon that will be placed next to panel label.
    */
   icon?: string | SVG;
```

Comment:
Please change this description in the website also:
![image](https://github.com/dxc-technology/halstack-react/assets/10975289/d2af2638-9829-4172-8793-47e3a43e8120)

Same for the accordion

Reply:
This will be updated in a separate PR.